### PR TITLE
I added action for the reducer 

### PR DESCRIPTION
**Pull request by ICrossfadE**
I added the action 'CREATE'. Which adds a new object to the array of objects

### CODE REVIEW

File: src/components/taskItem/taskItem.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
 import "./taskItem.css";
-import { state } from "../../state";
+import { store } from "../../state/index";
```
Comment (SASHAANGELM) [inline src/components/taskItem/taskItem.js:2]: No need to have /index here, `"../../state"` should be enough

File: src/components/Buttons/Buttons.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import ButtonCreateItem from "../buttonCreateItem/buttonCreateItem";
-import { state } from "../../state/index";
+import { store } from "../../state/index";
```
Comment (SASHAANGELM) [inline src/components/Buttons/Buttons.js:2]: No need to have /index here, "../../state" should be enough

File: src/components/buttonCreateItem/ButtonCreateItem.js [hunk 3]
```diff
@@ -1,12 +1,34 @@
...
+  const dispatch = useDispatch();
+  const taskList = useSelector((state) => state.taskList);
+
   const style = {
     backgroundColor: item.color,
   };
+
+  const createNewTask = (typeId) => {
+    const newTask = {
+      id: taskList.length + 1,
```
Comment (SASHAANGELM) [inline src/components/buttonCreateItem/ButtonCreateItem.js:13]: For new Id you need to have better way to get next id:
`Math.max(taskList.map(task => task.id)) + 1`

**Review by SASHAANGELM — CHANGES_REQUESTED** [review]

File: src/components/buttonCreateItem/ButtonCreateItem.js (hunk 3 above)
Comment (ICrossfadE) [reply src/components/buttonCreateItem/ButtonCreateItem.js:13]: In you answer return to NaN

I try a such variant (Math.max(...taskList.map((task) => task.id)) + 1) this option returns correct max id

**Review by SASHAANGELM — APPROVED** [review]

**Pull request merged**